README.md

```diff
@@ -98,12 +98,20 @@ Within your engine's `addon` directory, add a new `engine.js` file:
 ```js
 import Engine from 'ember-engines/engine';
 import Resolver from 'ember-engines/resolver';
+import loadInitializers from 'ember-load-initializers';
 
-export default Engine.extend({
-  modulePrefix: 'ember-blog',
+let Eng;
```

Comment:
Agreed, I was just blindly copying the format of the app/app.js blueprint. This is much better.